include/counter.h

```diff
@@ -97,7 +97,9 @@ static inline u32_t counter_read(struct device *dev)
  * @param dev Pointer to the device structure for the driver instance.
  * @param callback Pointer to the callback function. if this is NULL,
```

Comment:
While you're in here, can you also capitalize "If" at the beginning of this sentence.